sdk/eventhub/Azure.Messaging.EventHubs.Experimental/src/Azure.Messaging.EventHubs.Experimental.csproj

```diff
@@ -13,10 +13,10 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Messaging.EventHubs" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" />
-    <PackageReference Include="System.Reflection.TypeExtensions" />
-    <PackageReference Include="System.Threading.Channels" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Condition="'$(TargetFramework)' != 'net6.0'" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Condition="'$(TargetFramework)' != 'net6.0'" />
+    <PackageReference Include="System.Reflection.TypeExtensions" Condition="'$(TargetFramework)' != 'net6.0'" />
+    <PackageReference Include="System.Threading.Channels" Condition="'$(TargetFramework)' != 'net6.0'" />
     <PackageReference Include="System.Threading.Tasks.Extensions" />
   </ItemGroup>
```

Comment:
You might have to split this into 2 suggestions so I can apply it easier.

Reply:
Apologies, but I'm not quite sure what you're asking here.   I had to manually format the suggestion because it touches areas not in the change set.    I believe that you'd need to manually make the change locally.

Reply:
Did not need to do it manually, I carefully written a suggestion comment and then committed it.